packages/testring-cli-config/src/merge-configs.ts

```diff
@@ -0,0 +1,26 @@
+import * as deepmerge from 'deepmerge';
+import { IConfig } from '@testring/types';
+
+const emptyTarget = value => Array.isArray(value) ? [] : {};
+const clone = (value, options) => deepmerge(emptyTarget(value), value, options);
+
+function compressPlugins(target, source, options) {
+    const destination = target.slice();
```

Comment:
it's better to use `const destination = [...target]` or `const destination = Array.from(target)`